README.md

```diff
@@ -1,2 +1,275 @@
+
+![Domain Message Flow Modelling](resources/domain-message-flow.jpg)
+
+#### Tools
+- [Business Process Model and Notation](https://en.wikipedia.org/wiki/Business_Process_Model_and_Notation)
+- [Domain Message Flow Modelling](https://github.com/ddd-crew/domain-message-flow-modelling)
+- [Process Modelling EventStorming](https://www.eventstorming.com/)
+- [Sequence Diagrams](https://en.wikipedia.org/wiki/Sequence_diagram)
+
+#### Who to Involve
+- Domain Experts
+- Product Managers
+- Software Architects
+- Software Engineers
+
+### 5. Strategize
+Strategically map out your sub-domains to identify core domains: the parts of the domain which have the greatest potential for business differentiation or strategic significance.
+
+Time and resources are limited, so understanding which parts of the domain to focus on is critical to delivering optimal business impact. 
+
+By analysing what your core domains are, you will have a better idea of how much quality and rigour to each part of your system, and you'll be able to make highly-educated build vs buy vs outsource decisions.
+
+As a starting point, we recommend [Core Domain Charts](https://github.com/ddd-crew/core-domain-charts).
+
+![Core Domain Charts](resources/core-domain-chart.jpg)
+
+#### Tools
+- [Core Domain Charts](https://github.com/ddd-crew/core-domain-charts)
+- [Purpose Alignment Model](https://www.informit.com/articles/article.aspx?p=1384195&seqNum=2)
+- [Wardley Mapping](https://learnwardleymapping.com/)
+
+#### Who to Involve
+- Domain Experts
+- Business Stakeholders
+- Product Managers
+- Software Architects
+- Software Engineers
+- Technology Leadership
+
+### 6. Organise
+Organise autonomous teams that are optimised for fast flow and aligned with context boundaries.
+
+We need to organise teams to have autonomy, clear goals and sense of purpose. In order to do that we need to take into account organisational constraints, so that we can organise teams for fast flow. 
+
+We can optimise how people collaborate with each other if we align teams with context boundaries. In order to right-size the teams we need to take into account available talent, cognitive load, communication overhead, and bus factor. 
+
+As a starting point, we recommend visualising sociotechnial architecture with the [Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps).
+
+![Context Map](resources/context-map.jpg)*Credit: Michael Plöd*
+
+#### Tools
+- [Dynamic Reteaming](https://leanpub.com/dynamicreteaming) 
+- [Pioneers, Settlers & Town Planners](http://wardleypedia.org/mediawiki/index.php/Pioneers_settlers_town_planners)
+- [Team Topologies](https://teamtopologies.com/)
+- [Visualising Sociotechnical Architecture with Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps)
+
+#### Who to Involve
+- Human Resources
+- Software Architects
+- Product Managers
+- Senior Leadership
```

Comment:
Who is now senior leadership, compared with technical leadership? What have the architects to do with all this?  No "architect" title guarantees good leadership...